modelerfour/checker/checker.ts

```diff
@@ -39,7 +39,7 @@ export class Checker {
 
     const types = values(<Schema[]>this.codeModel.schemas.objects).concat(values(this.codeModel.schemas.groups)).concat(values(this.codeModel.schemas.choices)).concat(values(this.codeModel.schemas.sealedChoices)).toArray()
     for (const dupe of values(types).duplicates(each => each.language.default.name)) {
-      this.session.error(`Duplicate Object Schema '${dupe}' detected.`, []);
+      this.session.error(`Object schemas with '${dupe}' name  detected.`, []);
```

Comment:
Saying a duplicate was found seemed to be more helpful, now it just seems like an object schema was detected.